loom/script/runtime/lsLuaState.cpp

```diff
@@ -96,9 +97,11 @@ void LSLuaState::open()
     luaopen_math(L);
     luaL_openlibs(L);
 
+#ifdef LUAJIT_MODE_MASK
```

Comment:
Very professional looking hacky workaround. (Sincerely - it has comment + nice ifdef and it's clean overall.)